cdap-docs/admin-manual/source/security/impersonation.rst

```diff
@@ -47,7 +47,3 @@ The configured HDFS delegation token timeout must be longer than the configured
 partition duration (``stream.partition.duration``), which has a default value of
 one hour (3600000). It must also be larger than the log saver's maximum file
 lifetime (``log.saver.max.file.lifetime.ms``), which has a value of six hours (21600000).
-
```

Comment:
I believe there is one version of Hive (0.13, 0.14?) for which it still does not work